### PR TITLE
My first version of user, pls check

### PR DESCRIPTION
**Pull request by ghost**
Hi!
Please check my task

### CODE REVIEW

**Review by koics97 — APPROVED** [review]
Looks perfect!

File: webshop-backend/src/shopping/User.java [hunk 1]
```diff
@@ -2,4 +2,19 @@
 
 public class User {
 
+    private String name;
+    private Address address;
+
+    private int age;
+
```
Comment (lyancsie) [inline webshop-backend/src/shopping/User.java]: i always get confused seeing these random empty lines

File: webshop-backend/src/shopping/Country.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package shopping;
+
+public enum Country {
+ Hungary, UK, Germany
+}
```
Comment (lyancsie) [inline webshop-backend/src/shopping/Country.java]: Please make sure to follow the enum naming conventions

**Comment by deb05121** [comment]
I miss getters and setters.

File: webshop-backend/src/shopping/Address.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package shopping;
+
+public class Address {
+    Country country;
+    String city;
```
Comment (KJudit01) [inline webshop-backend/src/shopping/Address.java:5]: It is OK here.

**Review by deb05121 — APPROVED** [review]

**Review by lyancsie — APPROVED** [review]

**Pull request merged**